code/style.css

```diff
@@ -2,6 +2,327 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+
+
+/* for smooth scrolling when pressing the animation */
+html{
+    scroll-behavior: smooth;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    max-width: 100vw;
+}
+
+
+/* Hero image and text */
+.hero {
+    background: url('https://images.unsplash.com/photo-1621605815971-fbc98d665033?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80') center;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
```

Comment:
I love that your header and text and button is so smooth and responsive when changing the window-size! very pleasing.